.gitignore

```diff
@@ -20,6 +20,8 @@ server/docs
 # Ignore local Django settings
 *secrets.py
 *settings_secret.py
+*settings_custom.py
+*settings_overrides.py
```

Comment:
Since many of the files override others, what do you think about something like settings_local.py or settings_developer.py that, at a glance, might give more info about the file's purpose?

Still looking over the changes, but so far everything looks great!

Reply:
I support `settings_local`, personally. But can support `_dev(eloper)` just as easily.

> __Note__: We do have the `-dev` precedent set by `docker-compose-dev.all.debug.yml`—this could _later_ change to use similar semantics as is decided for the settings file (ex: `docker-compose_local.yml`).